db/db.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/lib/pq"
 	"github.com/packethost/pkg/log"
 	"github.com/pkg/errors"
+	migrate "github.com/rubenv/sql-migrate"
```

Comment:
I thought you said https://github.com/golang-migrate/migrate, goimports fail?

Reply:
no, from the community I got this library as a suggestion and it is very easy to embed, kind of cool!

Reply:
ah ok cool, just wondering.